### PR TITLE
Redmine 25926 Créer un webservice permettant d'exposer les coordonnées des lieux de rendez-vous

### PR DESCRIPTION
**Pull request by AnasBenjelloun**

### CODE REVIEW

File: pom.xml
Comment (FrancoisEricMerlin) [inline pom.xml:1]: you must declare version using open range  for all lutece dependencies:

sample lutece-core [7.0.8,)

File: src/java/fr/paris/lutece/plugins/appointment/modules/rest/service/AppointmentRestService.java
Comment (FrancoisEricMerlin) [inline src/java/fr/paris/lutece/plugins/appointment/modules/rest/service/AppointmentRestService.java:1]: why do you create  a service  for only call another service.
can you delete this service and call directly _appointmentSlotsService.getAvailableTimeSlotsAsList(search) in AppointmentRest

**Comment by FrancoisEricMerlin** [comment]
can you resolved conflicts and updating your pull request

**Comment by NLG85** [comment]
Please use a version range for all the dependencies : example for lutece core [7.0.9,) instead of 7.0.9

**Pull request merged**